src/main/java/uk/gov/hmcts/reform/sscs/controller/test/reminder/ReminderTestController.java

```diff
@@ -0,0 +1,89 @@
+package uk.gov.hmcts.reform.sscs.controller.test.reminder;
```

Comment:
Does this class need to be moved to src/test since it's only used in tests?

Reply:
Possibly - wasn't sure if it would work like that, but that might be better if it still works. Let me try.

Reply:
Fixed